README.md

```diff
@@ -5,6 +5,10 @@
 
 Natron is a free, open-source (GPLv2 license) video compositor, similar in functionality to Adobe After Effects, Foundry's Nuke, or Blackmagic Fusion. It is portable and cross-platform (GNU/Linux, macOS, and Microsoft Windows).
 
+![Hero Image of Natron](./MarkdownAssets/Natron-Hero-Image.png)
```

Comment:
+1 this is a README.md for the source repo. we don't need screenshots here

Reply:
Ok, if removing the image is your intention, then I'll remove it as well and keep it for use on the website instead.